docs/next/api/autoColumnSize.md

```diff
@@ -76,7 +76,7 @@ if (plugin.isEnabled()) {
 
 ### autoColumnSize
   
-::: source-code-link https://github.com/handsontable/handsontable/blob/develop/src/dataMap/metaManager/metaSchema.js#L2420
+::: source-code-link https://github.com/handsontable/handsontable/blob/develop/src/dataMap/metaManager/metaSchema.js#L2429
```

Comment:
The link points to the `strict` option. Should generate the link to line L2326 (https://github.com/handsontable/handsontable/blob/develop/src/dataMap/metaManager/metaSchema.js#L2326).

Can you check other source code links?

Reply:
These links link to the `develop` branch, we are on `feature/issue-7624`. That the purpose of diffreneces.

Reply:
I'm going to create an issue to use commit SHA instead of branch. I won't block merging this PR for now - I need it to finish to another PRs (are draft now)

Reply:
#8205